backend/src/main/java/ba/unsa/etf/si/bbqms/tenant_service/api/TenantService.java

```diff
@@ -0,0 +1,9 @@
+package ba.unsa.etf.si.bbqms.tenant_service.api;
+
+import ba.unsa.etf.si.bbqms.domain.Tenant;
+import ba.unsa.etf.si.bbqms.ws.models.AddTenantDto;
+
+public interface TenantService {
+    public Tenant addTenant(final AddTenantDto request);
+    public Tenant findByCode(final String code);
```

Comment:
`public` is redundant in interfaces. All methods are by default public.

Reply:
Removed `public` modifier